utils/runtime-fuzzer/fuzz/fuzz_targets/main.rs

```diff
@@ -44,21 +44,31 @@ fuzz_target!(|seed: u64| {
 // directory before running fuzz test.
 //
 // If directory already exists for the current run, it will be cleared.
-fn dump_seed(seed: u64) -> Result<(), String> {
+fn dump_seed(seed: u64) -> IoResult<()> {
     let seeds_file = RUN_INTIALIZED.get_or_try_init(|| {
         let seeds_dir = Path::new(SEEDS_STORE_DIR);
-        if seeds_dir.exists() {
-            fs::remove_dir_all(seeds_dir).map_err(|e| e.to_string())?;
+        match seeds_dir.exists() {
+            true => clear_dir(seeds_dir)?,
+            false => fs::create_dir_all(seeds_dir)?,
         }
-        fs::create_dir_all(seeds_dir).map_err(|e| e.to_string())?;
 
-        Ok::<_, String>(format!("{SEEDS_STORE_DIR}/{SEEDS_STORE_FILE}"))
+        Ok::<String, IoError>(format!("{SEEDS_STORE_DIR}/{SEEDS_STORE_FILE}"))
     })?;
 
     OpenOptions::new()
         .create(true)
         .append(true)
         .open(seeds_file)
         .and_then(|mut file| writeln!(file, "{seed}"))
-        .map_err(|e| e.to_string())
+}
+
+fn clear_dir(path: &Path) -> IoResult<()> {
+    for dir_entry in fs::read_dir(path)? {
+        let dir_entry = dir_entry?;
+        let path = dir_entry.path();
+
+        fs::remove_file(path)?;
```

Comment:
We can write it in one line I guess.
```suggestion
        fs::remove_file(dir_entry?.path())?;
```

Reply:
done.